pylint/config/arguments_manager.py

```diff
@@ -447,7 +447,10 @@ def generate_config(
         )
         options_by_section = {}
         sections = []
-        for group in self._arg_parser._action_groups:
+        for group in sorted(
+            self._arg_parser._action_groups,
+            key=lambda x: (x.title != "Main", x.title),
+        ):
```

Comment:
Shouldn't the options also be sorted line 460 to 464 ?

Reply:
👍🏻  Done, although it didn't change much. I force pushed the changes to keep the history clean for the merge commit.